app/models/agents/jira_agent.rb

```diff
@@ -16,8 +16,9 @@ class JiraAgent < Agent
       - `jira_url` specifies the full URL of the jira installation, including https://
       - `jql` is an optional Jira Query Language-based filter to limit the flow of events. See [JQL Docs](https://confluence.atlassian.com/display/JIRA/Advanced+Searching) for details. 
       - `username` and `password` are optional, and may need to be specified if your Jira instance is read-protected
+        - *Note*: Using your Jira `password` is no longer supported in Jira Cloud. You must use an [API token](https://id.atlassian.com/manage-profile/security/api-tokens) as your password.
       - `timeout` is an optional parameter that specifies how long the request processing may take in minutes.
-
+      - The issue's `changelog` is now returned with this Agent.
```

Comment:
Interesting idea to use the Agent description as a changelog. Not really sure how I feel about it, at first I didn't like it, but after thinking about it it kind of makes sense.
If we keep it, I think I would prefer it at the bottom of the description and add a date.